tests/lib/templates/reconciliationTexts.test.js

```diff
@@ -452,5 +452,22 @@ describe("ReconciliationText", () => {
 
       expect(result.text_parts).toEqual([{ name: "empty_part", content: "" }]);
     });
+
+    it("should exclude downloadable_as_docx and show warning when externally_managed is false", () => {
```

Comment:
This one looks good :+1: 
Maybe we can add a similar one testing that it's included when externally_managed is true?

Reply:
Positive scenario has been added as well